lib/superset/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Superset
-  VERSION = "0.2.5"
+  VERSION = "0.3.0"
```

Comment:
Looking at the Terminal Table Changelog: https://github.com/tj/terminal-table/blob/master/History.rdoc

We're crossing multiple major version boundaries with the jump from 1.8 to 4.0, but I'm not sure if it warrants a major version bump here.

The main changes are:

* Small formatting difference on table edges
* Dropped support for Ruby <= 2.3.x
    * We already require at least Ruby 2.7.8 in our gemspec
* Non-unicode characters are no longer supported and will be replaced

However, I think all of these changes are backwards compatible, as we're just using Terminal Table to output information in the console, and aren't parsing the results.

Keen to get a second opinion though.

Reply:
@SyntheticDave  lg +1

re  _"We're crossing multiple major version boundaries with the jump from 1.8 to 4.0, but I'm not sure if it warrants a major version bump here."_

Tend to agree .. only minimal output to terminal changes seem to not warrant the jump to 3.0.0
how about jump to VERSION = "0.2.6" ?

other than that .. I QA'd it a bunch as well .. so good to merge depending on the versioning

Reply:
Sounds good to me. I'm happy with just a patch bump.